autosubmit/autosubmit.py

```diff
@@ -39,6 +67,8 @@
 from .experiment.experiment_common import copy_experiment
 from .experiment.experiment_common import new_experiment
 from .git.autosubmit_git import AutosubmitGit
+from .history.experiment_history import ExperimentHistory
```

Comment:
I think if we will update/fix the imports here, we might as well make them uniform. At the moment we have relative and absolute imports. Let's make everything absolute (there are some corner cases where it may fail to start a debugger or run as a module in IDE/cmdline when you have these imports that start with a dot).